### PR TITLE
Fetch from database only if identifier is existing

### PR DESCRIPTION
**Pull request by staegi**

### CODE REVIEW

File: Lib/DoctrineObjectConstructor.php [hunk 1]
```diff
@@ -83,4 +86,3 @@ public function construct(VisitorInterface $visitor, ClassMetadata $metadata, $d
         return $object;
```
Comment (cgallardo) [inline Lib/DoctrineObjectConstructor.php]: $objectManager->initializeObject($object); shouldn't you also check if $object is null before calling this method?

Comment (staegi) [inline Lib/DoctrineObjectConstructor.php]: Not necessary because unit of work checks the type. See `Doctrine\ODM\MongoDB\UnitOfWork`

Comment (cgallardo) [inline Lib/DoctrineObjectConstructor.php]: I see, right there :)

Comment (cgallardo) [inline Lib/DoctrineObjectConstructor.php]: :+1:

**Pull request merged**